src/i18n/locales/de.json

```diff
@@ -1,7 +1,14 @@
 {
-  "navNewVocab": "Neue Vokabeln",
-  "navLearn": "Lernen",
-  "navProgress": "Fortschritt",
-  "navAllVocabulary": "Alle Vokabeln",
-  "navGroupLearning": "Gruppe lernen"
+  "global": {
+    "add": "Hinzufügen",
+    "none": "Keine"
+  },
+  "nav": {
+    "newVocab": "Neue Vokabeln",
+    "learn": "Lernen",
+    "progress": "Fortschritt",
+    "allVocabulary": "Alle Vokabeln",
+    "groupLearning": "Gruppe lernen",
+    "settings": "Einstellungen"
+  }
 }
```

Comment:
could you please delete the german translation from this pr? You can add it later on localazy if this pr is merged into the main branch.

Reply:
I've reverted my changes to the German translation file.